h2o-genmodel/src/main/java/hex/genmodel/tools/PrintMojo.java

```diff
@@ -21,10 +30,16 @@
   private static int treeToPrint = -1;
   private static int maxLevelsToPrintPerEdge = 10;
   private static boolean detail = false;
-  private static String outputFileName = null;
+  private String outputFileName = null;
   private static String optionalTitle = null;
   private static PrintTreeOptions pTreeOptions;
   private static boolean internal;
+  private String outputFormat = "dot";
+  private static final String tmpOutputFileName = "tmpOutputFileName.gv";
+
+  private LabeledVertexProvider vertexProvider = new LabeledVertexProvider();
```

Comment:
it looks like these are only needed when doing png exports and also are used in one method, do we realy need them as instance variables?